### ISSUE TITLE
shakespeare 2.0.20 doesn't compile with lts-13.0 on Windows

### ISSUE THREAD

**Issue by leohaskell** [yesodweb/shakespeare#233]
Steps to reproduce:
```haskel
git clone https://github.com/yesodweb/shakespeare.git
cd shakespeare
stack build --resolver=lts-13.0
```

I'd expect this to succeed, since `shakespeare` is included in `lts-13.0`, but it's stuck at
```Compiling Text.Internal.CssCommon```.

**Comment by snoyberg** [comment]
I believe this is a bug in GHC 8.6.3 itself. GHC 8.6.4 is slated to be released and should include a fix for this. For now, can you downgrade to LTS 12?

**Comment by leohaskell** [comment]
Yes, I can, but I think you should consider removing the package from stable hackage, until the issue is resolved.

**Comment by ygale** [comment]
@leohaskell You meant remove from stable stackage, right? It should not be removed from hackage - people not on Windows may still want to use it.

**Comment by leohaskell** [comment]
@ygale, Stackage means stable hackage : )
